### ISSUE TITLE
Temporary Directory does not get set when running log2timeline tool.

### ISSUE THREAD

**Issue by dc3-plaso** [log2timeline/plaso#1092]
**Plaso version:**

1.5.2_20161108

**Operating system Plaso is running on:**

Windows 7 64bit

**Installation method:**

dev environment

**Description of problem:**
The "--temporary_directory" argument that is advertised by the log2timeline tool doesn't actually get set. 
https://github.com/log2timeline/plaso/blob/master/tools/log2timeline.py#L655 is missing "temporary_directory=self._temporary_directory"

Also, this flag is a bit missleading. It seems like it should redirect the tmp* folders that get created by the storage file. I suggest rewording the help message to make this more clear and perhaps in the future allow the user to redirect these folders.

**Comment by joachimmetz** [comment]
@dc3-plaso this was overhauled since the 1.5.2_20161108 release. 

I'm keeping this open to the end of January 2018, let us know if this still not working as expected or what we can change in the option description.

This option is not intended to redirect the temporary paths used by the storage. Instead it is used for temporary files created during extraction, such as copies of SQLite databases.

**Comment by Onager** [comment]
Closing this out per previous comment.
